### PR TITLE
Single container per setup.py

### PR DESCRIPTION
**Pull request by kyocum**
1.) create one container per setup.py
2.) support `dsdt dockerize --get-id` to allow external programs to retag
3.) necessitates adding a required param to `dsdt run` to specify the path to the setup.py file.

### CODE REVIEW

File: disdat/dockerize.py [hunk 1]
```diff
@@ -116,42 +154,24 @@ def dockerize(pipeline_root,
...
-        touch_command = [
-            'touch',
-            odbc_file_path
-        ]
-        retval = _do_subprocess(touch_command, cli)
-        if retval: return retval
+    retval = _copy_in_dot_file(disdat_config, docker_context, "odbc.ini", "dot_odbc_ini_file", cli)
+    if retval: return retval
+
+    setup_file = os.path.join(pipeline_root, 'setup.py')
```
Comment (seanr15) [inline disdat/dockerize.py]: Probably should assert os.path.exists(setup_file)

File: disdat/run.py [hunk 2]
```diff
@@ -464,6 +457,8 @@ def _run(
 
     pfs = fs.DisdatFS()
 
+    pipeline_setup_file = os.path.join(pipeline_root, 'setup.py')
```
Comment (seanr15) [inline disdat/run.py]: Same as above

File: disdat/dockerize.py (hunk 1 above)
Comment (kyocum) [reply disdat/dockerize.py]: Added check

File: disdat/run.py (hunk 2 above)
Comment (kyocum) [reply disdat/run.py]: Added check

**Pull request merged**